src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/CornerRadius/CornerRadiusEffect.shared.cs

```diff
@@ -1,82 +1,47 @@
 ﻿using System;
-using System.ComponentModel;
-using System.Linq;
 using Xamarin.Forms;
 using Xamarin.Forms.Shapes;
 
 namespace Xamarin.CommunityToolkit.Effects
 {
-	public class CornerRadiusEffect : NullEffect
+	public class CornerRadiusEffect
 	{
 		public static readonly BindableProperty CornerRadiusProperty = BindableProperty.CreateAttached(
 			nameof(CornerRadius),
 			typeof(CornerRadius),
 			typeof(CornerRadiusEffect),
 			default(CornerRadius),
-			propertyChanged: TryGenerateEffect);
+			propertyChanged: OnCornerRadiusPropertyChanged);
 
-		public static CornerRadius GetCornerRadius(BindableObject? bindable)
-			=> (CornerRadius)(bindable?.GetValue(CornerRadiusProperty) ?? throw new ArgumentNullException(nameof(bindable)));
-
-		public static void SetCornerRadius(BindableObject? bindable, CornerRadius value)
-			=> bindable?.SetValue(CornerRadiusProperty, value);
-
-		static void TryGenerateEffect(BindableObject? bindable, object oldValue, object newValue)
+		static void OnCornerRadiusPropertyChanged(BindableObject bindable, object oldValue, object newValue)
 		{
-			if (bindable is not VisualElement elementView)
+			if (!(bindable is VisualElement elementView))
```

Comment:
when I did that code I didn't use C#9
```suggestion
			if (bindable is not VisualElement elementView))
```